app/index.tsx

```diff
@@ -1,5 +1,6 @@
 import { Redirect } from "expo-router";
 
 export default function Root() {
-    return <Redirect href="/(protected)/(tabs)/home" />;
+    return <Redirect href="/(protected)/chat/chat-list" />;
+    // return <Redirect href="/(protected)/(tabs)/home" />;
```

Comment:
pwede pa create lang button samwer else like sa profile siguro para ma reach parin ang other parts of the app after ma merge to na pr hehe